lib/src/widgets/movie_poster_detail.dart

```diff
@@ -19,11 +19,14 @@ class MoviePosterDetail extends StatelessWidget {
           height: MovieDimensions.paddingVerticalMedium,
         ),
         Center(
-          child: Image.network(
-            '${MovieStrings.gridBuilderImageUrlStart + imagePath}',
-            fit: BoxFit.fitWidth,
-            height: MovieDimensions.detailsImageHeight,
-            width: MovieDimensions.detailsImageWidth,
+          child: Hero(
+            tag: '$imagePath',
+            child: Image.network(
+              '${MovieStrings.gridBuilderImageUrlStart + imagePath}',
+              fit: BoxFit.fitWidth,
+              height: MovieDimensions.detailsImageHeight,
+              width: MovieDimensions.detailsImageWidth,
+            ),
           ),
         ),
```

Comment:
You are using the same structure here and in movies_grid_builder, can you abstract this Hero Widget into a new Widget, You can use the commons parameters that you use in both codes like tag, image network route, etc.
And for fit, height, width you can pass it as parameter the value that you want to use to.